### ISSUE TITLE
Nasty exception on `program.exe .`

### ISSUE THREAD

**Issue by hickford** [adrianaisemberg/CLAP#10]
`program.exe .` gives the nasty exception

```
Unhandled Exception: System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at CLAP.MultiParser.GetSingleTypeParser(String[] args, Object obj, ParserRegistration registration) in C:\Projects\CLAP\master\CLAP\MultiParser.cs:line 148
   at CLAP.MultiParser.RunTargets(String[] args, Object[] targets) in C:\Projects\CLAP\master\CLAP\MultiParser.cs:line 199
```

Whereas `program.exe -` is nicely handled.

**Comment by adrianaisemberg** [comment]
Cool!
Thanks. Will be fixed ASAP.

**Comment by adrianaisemberg** [comment]
Fixed in 3.8.
